samwise.js

```diff
@@ -1,4 +1,4 @@
-/**
```

Comment:
SCRIPT-This hardcoded value might be better suited as a constant at the top of the file.